### PR TITLE
WIP: Add script to update OCM manifest for Assisted Installer

### PR DESCRIPTION
**Pull request by lranjbar**
[MGMT-7283](https://issues.redhat.com/browse/MGMT-7283)

### CODE REVIEW

File: ci-operator/step-registry/edge/ci/ocm-publish/edge-ci-ocm-publish-commands.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+echo "Assisted Installer Image Info:\n"
+echo "IMAGE_NAME:     ${IMAGE_NAME}\n"
+echo "IMAGE_TAG:      ${IMAGE_TAG}\n"
+echo "IMAGE_VERSION:  ${IMAGE_VERSION}\n"
+echo "IMAGE_REGISTRY: ${IMAGE_REGISTRY}\n"
+echo "IMAGE_ORG:      ${IMAGE_ORG}\n"
+echo "------------------------------------------------------\n"
+echo "OCM Pipeline Info:\n"
```
Comment (celebdor) [inline ci-operator/step-registry/edge/ci/ocm-publish/edge-ci-ocm-publish-commands.sh:10]: All these `echo` that use \n` should be `printf` instead. Echo does not process backslash \n

**Comment by lranjbar** [comment]
/test/ pj-rehearse

File: ci-operator/step-registry/edge/ci/ocm-publish/edge-ci-ocm-publish-commands.sh [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+printf "OSCI_COMPONENT_VERSION: %s\n" ${OSCI_COMPONENT_VERSION}
+printf "OSCI_COMPONENT_SHA256: %s\n" ${OSCI_COMPONENT_SHA256}
+printf "OSCI_COMPONENT_REPO: %s\n" ${OSCI_COMPONENT_REPO}
+printf "OSCI_IMAGE_REMOTE_REPO: %s\n" ${OSCI_IMAGE_REMOTE_REPO}
+
+
+#cd /opt/build-harness/build-harness-extensions/modules/osci/
+#make osci/publish BUILD_HARNESS_EXTENSIONS_PATH=/opt/build-harness/build-harness-extensions
+
+export OSCI_PIPELINE_PRODUCT_PREFIX ?= release
```
Comment (pawanpinjarkar) [inline ci-operator/step-registry/edge/ci/ocm-publish/edge-ci-ocm-publish-commands.sh:66]: Why not 
`export OSCI_PIPELINE_PRODUCT_PREFIX="{OSCI_PIPELINE_PRODUCT_PREFIX:-release}"` :thinking:

Comment (lranjbar) [reply ci-operator/step-registry/edge/ci/ocm-publish/edge-ci-ocm-publish-commands.sh:66]: It was a copy pasta from the OCM scripts.. going to do a different thing though. See: https://github.com/open-cluster-management/build-harness-extensions/tree/master/modules/osci for the scripts this is running.

**Pull request closed without merging**